docs/layers/accounts/initialize-tfstate.mdx

```diff
@@ -35,15 +35,27 @@ If you look at `components/terraform/`, you'll see a bunch of directories. These
 
     Vendor the Terraform State Backend component by running the following command. The steps here require the `tfstate-backend` and `account-map` components.
 
-    <AtmosWorkflow workflow="vendor" fileName="baseline" />
+    :::tip What is Vendoring?
```

Comment:
this is the first time we ask them to vendor, so I thought this tip would be helpful